test/tools/utils.js

```diff
@@ -0,0 +1,291 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import {Float16Array} from '@petamoriken/float16';
+
+import {Tensor, sizeOfShape} from '../../src/lib/tensor.js';
+import {neg} from '../../src/unary.js';
+import {transpose} from '../../src/transpose.js';
+
+
+/**
+ * Convert data as required precision type.
+ * @param {Array<Number>} input
+ * @param {String} precisionType
+ * @return {(Array<Number>|Number)}
+ */
+function getPrecisionData(input, precisionType) {
+  let data;
+  const isNumber = typeof input === 'number';
+  if (isNumber) {
+    input = [input];
+  }
+
+  switch (precisionType) {
+    case 'float16':
+      data = new Float16Array(input);
```

Comment:
For browsers, would one need to launch the with `--js-flags="--js-float16array"` for this to work? (just asking - no action expected)

Reply:
Thanks for sharing this parameter! 👍 
I try it. Yes, it works.